catroid/src/main/java/org/catrobat/catroid/transfers/GetTagsTask.java

```diff
@@ -87,6 +91,18 @@ protected void onPostExecute(String response) {
 			tags.add(tagsJson.getString(i));
 		}
 
+		if (BuildConfig.CREATE_AT_SCHOOL) {
+			addTemplateTag(tags);
+		}
+
 		return Collections.unmodifiableList(tags);
 	}
+
+	private void addTemplateTag(List<String> tags) {
```

Comment:
since this is used only once: instead of making a separate method for this I'd put the code directly into line 95.